### PR TITLE
New example using the query param for Device Farm's list devices API

### PR DESCRIPTION
**Pull request by jamesknowsbest**
*Issue #, if available:*
NA

*Description of changes:*
Added an example of using the query param with Device Farm's list-devices API

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: awscli/examples/devicefarm/list-devices.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
+### To list all iOS private devices and their UDID value
+
+```bash
+aws devicefarm list-devices --region us-west-2 --query 'devices[?platform==`IOS` && fleetType==`PRIVATE`].[name, os, instances[].udid]'
```
Comment (justindho) [inline awscli/examples/devicefarm/list-devices.rst:4]: We can probably remove the `--region us-west-2` to maintain consistency between all the different service command/subcommand examples (we'll just assume that the user has already configured their region somewhere else, such as in their `~/.aws/config` or via an environment variable).

File: awscli/examples/devicefarm/list-devices.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
+### To list all iOS private devices and their UDID value
```
Comment (justindho) [inline awscli/examples/devicefarm/list-devices.rst:1]: Would you be able to format and phrase this file similar to how other rst example files are formatted? DynamoDB's `create-table.rst` example file could be a good reference: https://raw.githubusercontent.com/aws/aws-cli/develop/awscli/examples/dynamodb/create-table.rst

**Review by justindho — CHANGES_REQUESTED** [review]
Hi @jamesknowsbest, thanks for the contribution! Apologies for it taking so long to get back to you. I just had some comments regarding formatting/wording. Would you still be interested in pushing this PR forward to get it merged?

**Comment by tim-finnigan** [comment]
Closing as we have not heard back regarding changes requested on this PR.

**Pull request closed without merging**